javascript/javascript1/week4/voiceassistant.js

```diff
@@ -0,0 +1,81 @@
+let toDo = ["gym", "football", "coding", "reading"];
+let myName;
+const helloCommand = "hello my name is";
+let num1;
+let num2;
+
+function getReply(command) {
+  if (command.indexOf(helloCommand) === 0) {
+    myName = command.slice(helloCommand.length + 1);
+    return "nice to meet you " + myName;
+  } else if (command === "what is your name") {
+    return "your name is " + myName;
+  } else if (command === "add fishing to my to do") {
+    toDo.push("fishing");
+    console.log(toDo);
```

Comment:
You can make it dynamic, and accept any todo, not only fishing by replacing `add` and `to my todo` with empty string to get only the name of a todo item.
```js
  if (command.startsWith('add') && command.endsWith('to my todo')) {
    const todo = command.replace('add', '').replace(' to my todo', '');
    todos.push(todo);
    return `${todo} added to your todo`;
  }
```
for checking the command you can also use advanced way using regular expression
```js
/add.+to my todo/i.test(command);
```
`i` means `ignore case` to accept capital or lower case
`.+`: `.` means any characters `+` means `one or more`